src/DevBetterWeb.Web/Controllers/StripeWebhookHandler.cs

```diff
@@ -39,27 +40,37 @@ public class StripeWebhookHandler : Controller
     public async Task<IActionResult> Index()
     {
       var json = await new StreamReader(HttpContext.Request.Body).ReadToEndAsync();
-      _webhook.Content = $"{json}";
-      await _webhook.Send();
-
-      return Ok();
 
       try
       {
         var stripeEvent = EventUtility.ParseEvent(json);
         var stripeEventType = stripeEvent.Type;
 
-        if (stripeEventType.Equals("customer.subscription.created"))
+        if (stripeEventType.Equals("customer.subscription.created") || stripeEventType.Equals("customer.subscription.updated"))
```

Comment:
Use constants for these strings so they are only defined in one place in the entire source code.